include/NovelRT.Interop/Ecs/NrtSparseSetMemoryContainer.h

```diff
@@ -0,0 +1,92 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_NRTSPARSESETMEMORYCONTAINER_H
+#define NOVELRT_NRTSPARSESETMEMORYCONTAINER_H
+
+#include "../NrtInteropUtils.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    typedef struct SparseSetMemoryContainerHandle* NrtSparseSetMemoryContainer;
+
+    typedef struct SparseSetByteIteratorViewHandle* NrtSparseSetMemoryContainer_ByteIteratorView;
+
+    typedef struct SparseSetConstByteIteratorViewHandle* NrtSparseSetMemoryContainer_ConstByteIteratorView;
+
+    typedef struct SparseSetIteratorHandle* NrtSparseSetMemoryContainer_Iterator;
+
+    typedef struct SparseSetConstIteratorHandle* NrtSparseSetMemoryContainer_ConstIterator;
+
+NrtSparseSetMemoryContainer Nrt_SparseSetMemoryContainer_Create(size_t sizeOfDataTypeInBytes);
```

Comment:
Dunno how this got past formatting :)

Reply:
Just CLion things, but also, I've not been trying that hard since CLion disagrees with CI anyway lmao

Reply:
Ohhh CLion....